hack/kustomize-sub.sh

```diff
@@ -17,5 +17,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-root=$(dirname "${BASH_SOURCE[0]}")
-"$root/tools/bin/kustomize" build "$1" | "$root/tools/bin/envsubst"
\ No newline at end of file
+root=$(dirname "${BASH_SOURCE[0]}")/..
+kustomize="${root}/hack/tools/bin/kustomize"
+envsubst="${root}/hack/tools/bin/envsubst"
+
+make --directory="${root}" "${kustomize##*/}" "${envsubst##*/}"
+
+"${kustomize}" build "$1" | "${envsubst}"
```

Comment:
nit: new line

Reply:
Addressed via commit: e96caceecff07ec5a2ead5c5d56b5b769343125b

Reply:
Thanks for clarifying over messages. I force-pushed the changes with a new line added at EOF to `hack/kustomize.-sub.sh`. Please review again.